VideoLocker/res/menu/share_certificate.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:id="@+id/menu_item_share"
+        android:showAsAction="ifRoom"
+        android:title="@string/share_certificate_button" />
```

Comment:
Yes, it is a fat button, that's just what font awesome's share button looks like. We could alternatively use Google's which is thinner